test/test_manual.jl

```diff
@@ -43,6 +43,10 @@ isdir(outdir) && rm(outdir, recursive=true)
     @test Trixi2Vtk.pvd_filenames("", "manual", "out") == (joinpath("out", "manual"), joinpath("out", "manual_celldata"))
     @test_throws ErrorException Trixi2Vtk.pvd_filenames(("a", "b"), nothing, "out")
   end
+
+  @timed_testset "expand_filename_patterns" begin
+    @test Trixi2Vtk.expand_filename_patterns(["/*"]) isa Vector{String}
```

Comment:
Can you create a real unit test instead of this smoke test? Alternatively, can we use a pattern with an absolute path (and a relative path, if not already present) pattern for `trixi2vtk`?

Reply:
I tried to improve it in 9716fd3. Let me know what you think or if you have another idea.

Reply:
Looks much better, thanks :+1: